packages/records/lib/models/records.integration.test.ts

```diff
@@ -0,0 +1,179 @@
+import { expect, describe, it, beforeAll, afterEach } from 'vitest';
+import dayjs from 'dayjs';
+import * as uuid from 'uuid';
+import { migrate } from '../db/migrate.js';
+import { RECORDS_TABLE } from '../constants.js';
+import { db } from '../db/client.js';
+import * as Records from '../models/records.js';
+import { formatRecords } from '../helpers/format.js';
+import type { UnencryptedRecordData } from '../types.js';
+import type { UpsertSummary } from '../types.js';
+import { isErr } from '@nangohq/shared';
```

Comment:
my goal is to move `Result` to `@nangohq/utils` so records doesn't have to depend on `shared`